src/main/java/aquality/appium/mobile/application/MobileModule.java

```diff
@@ -22,5 +25,6 @@ protected void configure() {
         bind(ILocalServiceSettings.class).to(getLocalServiceSettingsImplementation()).in(Singleton.class);
         bind(IConfiguration.class).to(getConfigurationImplementation());
         bind(IElementFactory.class).to(getElementFactoryImplementation());
+        bind(IScreenFactoryProvider.class).to(ScreenFactoryProvider.class);
```

Comment:
based on our current structure you should add a method getScreenFactoryProviderImplementation() in some module.
it is done to have a possibility to override implemetation of IScreenFactoryProvider in DI container.